src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/config.json

```diff
@@ -0,0 +1,126 @@
+﻿{
+  "properties": {
+    "OSGroup": {
+      "explanation": "Target OS for result binaries.",
+      "type": "property",
+      "values": "Windows_NT, Linux, OSX, FreeBSD, NetBSD",
+      "default": "Windows_NT"
+     },
+    "RestoreDuringBuild": {
+      "explanation": "Enables/Disables the restoration of packages.",
+      "type": "property",
+      "values": "True, False",
+      "default": true
+    },
+    "BuildTests": {
+      "explanation": "Enables/Disables building tests against the product binaries.",
+      "type": "property",
+      "values": "True, False",
+      "default": true
+    },
+    "BuildPackages": {
+      "explanation": "Enables/Disables building packages.", 
+      "type": "property",
+      "values": "True, False",
+      "default": true
+    },
+    "BuildTestsAgainstPackages": {
+      "explanation": "Allows to build tests against product packages.",
+      "type": "property",
+      "values": "True, False",
+      "default": false
+    },
+    "BatchRestorePackages": {
+      "explanation": "MsBuild target that restores the packages.",
+      "type": "target",
+      "values": "",
+      "default": ""
+    },
+    "BatchGenerateTestProjectJsons": {
+      "explanation": "MsBuild target that generates the project Jsons files to build tests against packages.",
+      "type": "target",
+      "values": "",
+      "default": ""
+    }
+  },
+  "commands": {
+    "simple": {
+      "explanation": "Only builds managed product binaries. It doesn't restore packages.",
+      "processName": "msbuild",
+      "project": "build.proj",
```

Comment:
It is, but buid.proj is not always the entry point. As you said in another comment, we could change the `project` value for individual projects if desired (is not yet implemented though...)